src/main/java/edu/kis/powp/jobs2d/drivers/adapter/AbstractDriverAdapter.java

```diff
@@ -0,0 +1,24 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.legacy.drawer.shape.LineFactory;
+import edu.kis.powp.jobs2d.AbstractDriver;
+import edu.kis.powp.jobs2d.features.DrawerFeature;
+
+public class AbstractDriverAdapter extends AbstractDriver {

```

Comment:
The name does not indicate the purpose of the adapter or what was actually adapted.